metadata-ingestion/tests/integration/openapi/openapi_mces_golden.json

```diff
@@ -1,153 +1,249 @@
 [
 {
-    "proposedSnapshot": {
-        "com.linkedin.pegasus2avro.metadata.snapshot.DatasetSnapshot": {
-            "urn": "urn:li:dataset:(urn:li:dataPlatform:OpenApi,test_openapi.root,PROD)",
-            "aspects": [
-                {
-                    "com.linkedin.pegasus2avro.dataset.DatasetProperties": {
-                        "customProperties": {},
-                        "description": "List API versions",
-                        "tags": []
-                    }
-                },
-                {
-                    "com.linkedin.pegasus2avro.common.GlobalTags": {
-                        "tags": []
-                    }
-                },
-                {
-                    "com.linkedin.pegasus2avro.common.InstitutionalMemory": {
-                        "elements": [
-                            {
-                                "url": "https://raw.githubusercontent.com/OAI/learn.openapis.org/refs/heads/main/examples/",
-                                "description": "Link to call for the dataset.",
-                                "createStamp": {
-                                    "time": 1586847600,
-                                    "actor": "urn:li:corpuser:etl"
-                                }
-                            }
-                        ]
-                    }
-                },
+    "entityType": "dataset",
```

Comment:
I did visual validation that the metadata here is unchanged (save subtypes) and just re-organized.